src/dropdown/dropdown.js

```diff
@@ -115,6 +116,8 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         self.dropdownMenu.remove();
       });
     }
+
+    $element.on('keydown', keybindFilter);
```

Comment:
Ah, yes I guess you are right since the element would be getting removed from the DOM when the scope is being destroyed. Still interested on your thoughts about which PR implementation to use though.